File: algorithms/FooBar/fooBar.lua
function main()
    start = 1
    foo = 5
    bar = 7
    stop = 100
    num = 0
    fooString = "Foo"
    barString = "Bar"
    
    for num = start, stop, 1 do
        if num % foo == 0 & num % bar == 0 then
            print(fooString + barString)
        elseif num % foo == 0 then
            print(fooString)
        elseif num % bar == 0 then
            print(barString)
        else
            print(num)
        end
    end
    return 0
end

main()

File: algorithms/FooBar/fooBar.js
let start, foo, bar, stop, num, fooString, barString;

start = 1;
foo = 5;
bar = 7;
stop = 100;
fooString = "Foo";
barString = "Bar";

for (num = start; num <= stop; num++) {
    if (num % foo == 0 && num % bar == 0) {
        console.log(fooString + barString);
    } else if (num % foo == 0) {
        console.log(fooString);
    } else if (num % bar == 0) {
        console.log(barString);
    } else {
        console.log(num);
    }
}
